### DISCUSSION TITLE
Using a theme file generated from styled-dictionary with vanilla-extract

### DISCUSSION THREAD

**Discussion by hayawata3626** [vanilla-extract-css/vanilla-extract#1169] [Q&A]
I am uncertain about whether to use `@vanilla-extract/css` for the following scenario.

Currently, we are distributing design tokens using [style-dictionary](https://github.com/amzn/style-dictionary), which generates outputs like CSS variables and `theme.ts`. Below is a portion of the `theme.ts` file.

<img width="628" alt="スクリーンショット 2023-08-22 10 34 01" src="https://github.com/vanilla-extract-css/vanilla-extract/assets/15213369/9830f3a6-da6f-41e2-b430-c60c31b0e530">

When attempting to use the generated theme by passing it as an argument to `createGlobalTheme`, I encounter the following type error:

```
Argument of type '{ readonly font: { readonly title: { readonly l: { readonly fontSize: 32; readonly textDecoration: "none"; readonly fontFamily: "Noto Sans JP"; readonly fontWeight: 400; readonly fontStyle: "normal"; readonly fontStretch: "normal"; ... 4 more ...; readonly textCase: "none"; }; readonly m: { ...; }; readonly s: { ......' is not assignable to parameter of type 'Tokens'.
```

Error at the relevant line:
```ts
export const vars = createGlobalTheme(':root', theme);
```

What is the reason createGlobalTheme does not accept number types, etc.?

**Comment by hayawata3626** [comment]
This may be same with https://github.com/vanilla-extract-css/vanilla-extract/discussions/1040
So I will close this discussion.

**Reply by ancashoria** [reply]
Hey, did you manage to solve this?
